docs/xcm/building-with-xcm/xc-remote-transact.md

```diff
@@ -133,7 +121,7 @@ For example, let's assume you have a contract deployed on Shibuya and are callin
 | Caller Multilocation | `{ parents: 1, interior: X2 ( Parachain(2000), AccountId32 {network: NetworkId::Any, id: 0xd219f...f119 } ) }` |
 | Derived Account32Hash Address | `5FrhDFydxUwbWyXT1XDBhRUUYpQtiJJ6skB6n2XV4NubC9fP` |
 
-This means that the instructions like `WithdrawAsset` and `Transact` will be executed on remote chain as if origin was the address `5FrhDFydxUwbWyXT1XDBhRUUYpQtiJJ6skB6n2XV4NubC9fP`.
+This means that the following instructions like `WithdrawAsset` and `Transact` will be executed as if origin was address `5FrhDFydxUwbWyXT1XDBhRUUYpQtiJJ6skB6n2XV4NubC9fP`.
```

Comment:
They're not _following_, they have been covered in the previous chapter.

Reply:
done